cdap-api-common/src/main/java/co/cask/cdap/api/common/Bytes.java

```diff
@@ -257,6 +257,20 @@ public static String toHexString(byte [] bytes) {
   }
 
   /**
+   * Returns a byte[] by decoding from the given hexadecimal string. The string must be have an even length.
+   */
+  public static byte[] fromHexString(String str) {
```

Comment:
That's a good point. We currently don't have any unit-test covering the `Bytes` class at all. Will add it.